Tutorials/G-CNN_Honeycomb.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<blockquote>Based on the standard deviation, it looks like our wavefunction has on the order of 1% error. Not bad for 20 minutes of training!</blockquote><p> </p><p>To make a statement like this you should probably use the <code>error_of_mean</code> statistic, which is not the same as <code>sqrt(variance).</code> For example, the error of the mean goes to zero when the MC sample sizes becomes infinite, the variance will not (except in an exact eigenstate) since it has a physically meaningful value.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Okay, sorry, I guess <code>error_of_mean</code> is also not  really what we want here.</p><p> </p><p>Still, while using the variance as a measure of "distance to an eigenstate" is appropriate, I still think "1% of error" a bit misleading. We do not actually know, a priori, if we are close to the ground state or a local minimum. Not sure what the best way to phrase this is, though.</p>

Reply:
<p>Yeah, you’re right that this is probably a statement I can’t make. The correct way to do this is to see how the energy scales with variance. I could actually try that, it would be neat to look at anyway. It might make this section require a couple hours on a GPU but people can always hop in or out when they like</p>

Reply:
Yes, the variance scaling sounds like a good way to check this. If that's too much effort I think it's also fine to look at the variance, just with a bit more cautious language (pointing out that it indicates closeness to an eigenstate).